index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <title>Sum from 0 to N</title>
+</head>
+<body>
+    <div class="main">
+        <div class="sum-numbers">
+            <h2>Sum from 0 to N</h2>
+            <label for="numberInput">Enter a positive integer:</label>
+            <input type="number" id="numberInput" min="0">
+            <button onclick="calculateSum()">Calculate</button>
+            <p id="result"></p>
+        </div>
+        <div class="detect-even-odd">
+            <h2>Even and Odd numbers</h2>
+            <label for="numberInput1">Enter a positive integer:</label>
+            <input type="number" id="numberInput1" min="0">
+            <button onclick="evenoddnumbers()">Detect</button>
+            <p id="result1"></p>
+        </div>
+        <div class="staircase">
+            <h2>Simple Stair Case</h2>
+            <p>Please put a number between 2 to 100</p>
+            <div>
+              <label for="numberInput2">Enter a number:</label>
+              <input type="number" id="numberInput2" min="2" max="100">
+              <button onclick="generateStaircase()">Stair</button>
+            </div>
+            <p id="errorMessage"></p>
+            <div id="stairResult"></div>
+        </div>
+        <div class="print-numbers">
+            <h2>Display Numbers 1 to 100</h2>
+            <button onclick="numbers1to100()">Show Numbers</button>
+            <div id="result3"></div>
+        </div>
+        <div class="guessing-game">
+            <h2>Guessing Game</h2>
+            <label for="numberInput4">Which number between 1 to 100 is neither a prime number nor a composite number?</label>
+            <div>
+              <input type="number" id="numberInput4" min="1" max="100">
+              <button onclick="guessingGame()">Guess</button>
+            </div>
+            <p id="result4"></p>
+        </div>
+    </div>
+  <script>
+    let Experience = prompt("How long have you been in Rebase Academy? (In terms of years)")
+        if (Experience<0){
+            alert(1)
+        }
+        else if (Experience>0){
+            alert(1)
+        }
+        else{
+            alert(0)
+        }
+        let x=0;
+        while (x<100) {
+            console.log(x+1)
+            x++
+        }
+    function calculateSum() {
+            const input = document.getElementById('numberInput').value;
+            const n = parseInt(input);
+            if (isNaN(n) || n < 0) {
+                document.getElementById('result').textContent = 'Please enter a valid non-negative integer.';
+                return;
+            }
+            let sum = 0;
+            for (let i = 0; i <= n; i++) {
+                sum += i;
+            }
+            if (sum % 15 === 0) {
```

Comment:
Looking at the question for this, your results should be something like this:
```
1
2
Fizz
4
Buzz
Fizz
7
8
Fizz
Buzz
11
Fizz
13
14
FizzBuzz
```
and so on and so forth, right up to 100